src/atc/transformers/union_transformer.py

```diff
@@ -0,0 +1,18 @@
+from pyspark.sql import DataFrame
+
+from atc.etl import Transformer
+from atc.etl.types import dataset_group
+
+
+class UnionTransformer(Transformer):
+    """Returns the union of all input dataframes."""
+
+    def process(self, df: DataFrame) -> DataFrame:
+        return df
+
+    def process_many(self, datasets: dataset_group) -> DataFrame:
```

Comment:
what if something earlier in the ETL has made one of the dataset = NONE. Then it cannot union. Consider something like this:
```
for df in datasets:
        if df.values() is None:
            raise NoTableException(f"The {df.key} table dataframe in the list is None.")
```